spec/spec_helper.rb

```diff
@@ -1,25 +1,22 @@
 # encoding: utf-8
 
-if RUBY_ENGINE.eql?('ruby')
```

Comment:
__TODO:__ Check if this is still required for jruby. If it is, then restore it and make sure jruby is added back to the matrix.